src/app/modules/login-module/login/login.component.scss

```diff
@@ -0,0 +1,47 @@
+.main-container {
+  width: 100%;
+  min-height: 90vh;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: #ffffff;
```

Comment:
Please create file `colors.scss` in folder `scss/constants` on module level and create scss variables for colors.

Reply:
done